web/themes/custom/pineapple/sass/pineapple.styles.scss

```diff
@@ -8,3 +8,4 @@
 @import "base/__base.scss";
 @import "layouts/__layouts.scss";
 @import "components/__components.scss";
+@import 'components/field/__field.scss';
```

Comment:
This line and the file __field.scss file need to be deleted

Reply:
done